cytoscape-panzoom.js

```diff
@@ -191,10 +191,14 @@ SOFTWARE.
           // functions for calculating panning
           ////////////////////////////////////
 
+          function isMouseEvent(e) {
+            return e.type.substring(0, 5) === 'mouse';
+          }
+
           function handle2pan(e){
             var v = {
-              x: e.originalEvent.pageX - $panner.offset().left - $panner.width()/2,
-              y: e.originalEvent.pageY - $panner.offset().top - $panner.height()/2
+              x: (isMouseEvent(e) ? e.originalEvent.pageX : e.originalEvent.touches[0].pageX) - $panner.offset().left - $panner.width()/2,
```

Comment:
Probably not critical here but try to make touch/event stuff as efficient as possible; storing isMouseEvent(e) as a var and then using a single if clause is less evaluations than 2 ternaries. (though looking at the existing code, they aren't being terribly efficient anyway so again probably not too important)